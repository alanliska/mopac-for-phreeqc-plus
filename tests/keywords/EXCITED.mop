RHF SINGLET OPEN(2,2) ROOT=2
comment1
comment2
   C       -0.91179                    0.11035                    -0.00567           
   C       -0.96839                    1.44322                     0.06186           
   H       -0.06555                    2.04462                     0.02330            
   H       -1.91868                    1.95916                     0.15703            
   H       -1.56780                   -0.32846                    -0.69943            
   H       -0.23460                   -0.52753                     0.45062            

RHF EXCITED
comment1
comment2
   C       -0.91179                    0.11035                    -0.00567           
   C       -0.96839                    1.44322                     0.06186           
   H       -0.06555                    2.04462                     0.02330            
   H       -1.91868                    1.95916                     0.15703            
   H       -1.56780                   -0.32846                    -0.69943            
   H       -0.23460                   -0.52753                     0.45062            
