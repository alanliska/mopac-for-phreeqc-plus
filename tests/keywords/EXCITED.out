 *******************************************************************************
 **                                                                           **
 **                              MOPAC v22.1.0                                **
 **                                                                           **
 *******************************************************************************
 **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **
 **    Visit the DOI location for information on how to cite this program.    **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Feb  7 15:13:00 2024  *
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 2 ELECTRONS IN 2 LEVELS
 *  ROOT       - IN A C.I. CALCULATION, ROOT "2" TO BE OPTIMIZED.
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
RHF SINGLET OPEN(2,2) ROOT=2
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C         -0.91179000  *   0.11035000  *  -0.00567000  *
     2       C         -0.96839000  *   1.44322000  *   0.06186000  *
     3       H         -0.06555000  *   2.04462000  *   0.02330000  *
     4       H         -1.91868000  *   1.95916000  *   0.15703000  *
     5       H         -1.56780000  *  -0.32846000  *  -0.69943000  *
     6       H         -0.23460000  *  -0.52753000  *   0.45062000  *



          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C         -0.9118      0.1103     -0.0057
     2         C         -0.9684      1.4432      0.0619
     3         H         -0.0655      2.0446      0.0233
     4         H         -1.9187      1.9592      0.1570
     5         H         -1.5678     -0.3285     -0.6994
     6         H         -0.2346     -0.5275      0.4506

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   D2  


 SINGLET STATE CALCULATION

      THERE ARE    5 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 5


                       NO. OF SINGLY OCCUPIED LEVELS =    2

          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    65.481 HEAT:  115.3758
 CYCLE:     2 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:   112.268 HEAT:  111.0887
 CYCLE:     3 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    88.146 HEAT:  109.5028
 CYCLE:     4 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    43.313 HEAT:  106.3676
 CYCLE:     5 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    39.059 HEAT:  103.7754
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    48.641 HEAT:  100.3109
 CYCLE:     7 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    28.094 HEAT:  98.32571
 CYCLE:     8 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    15.878 HEAT:  98.20012
 CYCLE:     9 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    12.619 HEAT:  98.03829
 CYCLE:    10 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    13.091 HEAT:  97.93168
 CYCLE:    11 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    11.603 HEAT:  97.76892
 CYCLE:    12 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     5.167 HEAT:  97.61709
 CYCLE:    13 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     4.101 HEAT:  97.55076
 CYCLE:    14 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     3.628 HEAT:  97.50945
 CYCLE:    15 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.589 HEAT:  97.48910
 CYCLE:    16 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.677 HEAT:  97.48623

     GRADIENT =  0.67738 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF SINGLET OPEN(2,2) ROOT=2
 comment1
 comment2


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC v22.1.0 MacOS
                                                       Wed Feb  7 15:13:00 2024




          FINAL HEAT OF FORMATION =         97.48601 KCAL/MOL =     407.88147 KJ/MOL


          COSMO AREA              =         70.94 SQUARE ANGSTROMS
          COSMO VOLUME            =         52.70 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.67738          =       0.27654 PER ATOM
          NO. OF FILLED LEVELS    =          5
          AND NO. OF OPEN LEVELS  =          2
          MOLECULAR WEIGHT        =         28.0536         POINT GROUP:  C2v 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     6    H     4     2.90617
            H     5    H     3     2.22315
            H     4    H     5     1.21888


          SCF CALCULATIONS        =         20

          WALL-CLOCK TIME         =      0.086 SECONDS
          COMPUTATION TIME        =      0.067 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C         -0.92045770  *   0.10919361  *  -0.04749304  *
     2       C         -0.96381963  *   1.42314896  *   0.06827288  *
     3       H         -0.25689414  *   2.17072543  *  -0.23420271  *
     4       H         -1.71610695  *   2.06113285  *   0.49143033  *
     5       H         -1.31019268  *  -0.44686745  *  -0.91810339  *
     6       H         -0.48929192  *  -0.56392206  *   0.71463447  *

                             CARTESIAN COORDINATES

   1    C       -0.920457700     0.109193608    -0.047493038
   2    C       -0.963819631     1.423148964     0.068272878
   3    H       -0.256894142     2.170725435    -0.234202706
   4    H       -1.716106946     2.061132848     0.491430335
   5    H       -1.310192677    -0.446867446    -0.918103387
   6    H       -0.489291919    -0.563922056     0.714634466


           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -26.44306 -20.45529 -14.70732 -14.44061 -14.07416  -4.30123  -4.19490   5.75148
   5.87616   5.89989   6.41258   6.64601


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.494615        3.5054     1.09198     2.41341
    2          C          -1.155035        5.1550     1.07701     4.07803
    3          H           0.216692        0.7833     0.78331
    4          H           0.216846        0.7832     0.78315
    5          H           0.113419        0.8866     0.88658
    6          H           0.113463        0.8865     0.88654
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.125    -3.781    -0.334     3.797
 HYBRID        -0.004     0.128     0.010     0.128
 SUM            0.121    -3.653    -0.324     3.669


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.09198   0.42618   1.03462   0.95261
    2  C   1.07701   1.27791   1.03501   1.76511
    3  H   0.78331
    4  H   0.78315
    5  H   0.88658
    6  H   0.88654
 *******************************************************************************
 **                                                                           **
 **                              MOPAC v22.1.0                                **
 **                                                                           **
 *******************************************************************************
 **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **
 **    Visit the DOI location for information on how to cite this program.    **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Feb  7 15:13:00 2024  *
 *  EXCITED    - FIRST EXCITED STATE IS TO BE OPTIMIZED
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
RHF EXCITED
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C         -0.91179000  *   0.11035000  *  -0.00567000  *
     2       C         -0.96839000  *   1.44322000  *   0.06186000  *
     3       H         -0.06555000  *   2.04462000  *   0.02330000  *
     4       H         -1.91868000  *   1.95916000  *   0.15703000  *
     5       H         -1.56780000  *  -0.32846000  *  -0.69943000  *
     6       H         -0.23460000  *  -0.52753000  *   0.45062000  *



          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C         -0.9118      0.1103     -0.0057
     2         C         -0.9684      1.4432      0.0619
     3         H         -0.0655      2.0446      0.0233
     4         H         -1.9187      1.9592      0.1570
     5         H         -1.5678     -0.3285     -0.6994
     6         H         -0.2346     -0.5275      0.4506

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   D2  


 SYSTEM IS A BIRADICAL


 EXCITED STATE CALCULATION

      THERE ARE    5 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 5


                       NO. OF SINGLY OCCUPIED LEVELS =    2

          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    65.481 HEAT:  115.3758
 CYCLE:     2 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:   112.268 HEAT:  111.0887
 CYCLE:     3 TIME:   0.008 TIME LEFT:  2.00D  GRAD.:    88.146 HEAT:  109.5028
 CYCLE:     4 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    43.313 HEAT:  106.3676
 CYCLE:     5 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    39.059 HEAT:  103.7754
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    48.641 HEAT:  100.3109
 CYCLE:     7 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    28.094 HEAT:  98.32571
 CYCLE:     8 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    15.878 HEAT:  98.20012
 CYCLE:     9 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    12.619 HEAT:  98.03829
 CYCLE:    10 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    13.091 HEAT:  97.93168
 CYCLE:    11 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:    11.603 HEAT:  97.76892
 CYCLE:    12 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     5.167 HEAT:  97.61709
 CYCLE:    13 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     4.101 HEAT:  97.55076
 CYCLE:    14 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     3.628 HEAT:  97.50945
 CYCLE:    15 TIME:   0.004 TIME LEFT:  2.00D  GRAD.:     1.589 HEAT:  97.48910
 CYCLE:    16 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.677 HEAT:  97.48623

     GRADIENT =  0.67738 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF EXCITED
 comment1
 comment2


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC v22.1.0 MacOS
                                                       Wed Feb  7 15:13:00 2024




          FINAL HEAT OF FORMATION =         97.48601 KCAL/MOL =     407.88147 KJ/MOL


          COSMO AREA              =         70.94 SQUARE ANGSTROMS
          COSMO VOLUME            =         52.70 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.67738          =       0.27654 PER ATOM
          NO. OF FILLED LEVELS    =          5
          AND NO. OF OPEN LEVELS  =          2
          MOLECULAR WEIGHT        =         28.0536         POINT GROUP:  C2v 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     6    H     4     2.90617
            H     5    H     3     2.22315
            H     4    H     3     1.21888


          SCF CALCULATIONS        =         20

          WALL-CLOCK TIME         =      0.141 SECONDS
          COMPUTATION TIME        =      0.121 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C         -0.92045770  *   0.10919361  *  -0.04749304  *
     2       C         -0.96381963  *   1.42314896  *   0.06827288  *
     3       H         -0.25689414  *   2.17072543  *  -0.23420271  *
     4       H         -1.71610695  *   2.06113285  *   0.49143033  *
     5       H         -1.31019268  *  -0.44686745  *  -0.91810339  *
     6       H         -0.48929192  *  -0.56392206  *   0.71463447  *

                             CARTESIAN COORDINATES

   1    C       -0.920457700     0.109193608    -0.047493038
   2    C       -0.963819631     1.423148964     0.068272878
   3    H       -0.256894142     2.170725435    -0.234202706
   4    H       -1.716106946     2.061132848     0.491430335
   5    H       -1.310192677    -0.446867446    -0.918103387
   6    H       -0.489291919    -0.563922056     0.714634466


           Empirical Formula: C2 H4  =     6 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -26.44306 -20.45529 -14.70732 -14.44061 -14.07416  -4.30123  -4.19490   5.75148
   5.87616   5.89989   6.41258   6.64601


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C           0.494615        3.5054     1.09198     2.41341
    2          C          -1.155035        5.1550     1.07701     4.07803
    3          H           0.216692        0.7833     0.78331
    4          H           0.216846        0.7832     0.78315
    5          H           0.113419        0.8866     0.88658
    6          H           0.113463        0.8865     0.88654
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.125    -3.781    -0.334     3.797
 HYBRID        -0.004     0.128     0.010     0.128
 SUM            0.121    -3.653    -0.324     3.669


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.09198   0.42618   1.03462   0.95261
    2  C   1.07701   1.27791   1.03501   1.76511
    3  H   0.78331
    4  H   0.78315
    5  H   0.88658
    6  H   0.88654

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.23 SECONDS

 == MOPAC DONE ==
